pkg/operator/deploy/staticresources/master/deployment.yaml.tmpl

```diff
@@ -36,6 +36,13 @@ spec:
         ports:
         - containerPort: 8080
           name: http
+        resources:
+          limits:
+            cpu: 50m
```

Comment:
CPU limits are generally considered bad practice, since CPU is a compressible resource. Limits tend to harm way more than they help. Here's one of many articles explaining this further: https://medium.com/directeam/kubernetes-resources-under-the-hood-part-3-6ee7d6015965